network-api/networkapi/wagtailpages/models.py

```diff
@@ -517,6 +520,63 @@ class NewsPage(PrimaryPage):
     template = 'wagtailpages/static/news_page.html'
 
 
+class BlogPageTag(TaggedItemBase):
+    content_object = ParentalKey('wagtailpages.BlogPage', on_delete=models.CASCADE, related_name='tagged_items')
```

Comment:
Is `CASCADE` safe? E.g. are BlogPageTags unique to a single BlogPage? If not, and they're shared, then CASCADE is probably the wrong delete policy

Reply:
This code snippet is directly copied from http://docs.wagtail.io/en/v2.0/reference/pages/model_recipes.html#tagging . @Pomax @alanmoo can you provide code suggestion here as I don't know what appropriate value to put here instead.

Reply:
Playing with the review app, this seems to work as expected- adding the same tag to multiple posts, then removing it from one leaves it on the others.